### DISCUSSION TITLE
Kanban view

### DISCUSSION THREAD

**Discussion by lordsfantasy** [carbon-design-system/carbon#9753] [Ideas]
Any plans to implement a kanban view and provide OOB support to convert table view into kanban and kanban into a table?

**Comment by tw15egan** [comment]
No plans at the moment for any feature like this

**Comment by tay1orjones** [comment]
Hi @lordsfantasy - I'll convert this to a discussion to encourage others to respond. We'd be interested to see some research and info to back this request from yours and others' teams.
